upup/pkg/fi/cloudup/awstasks/vpc.go

```diff
@@ -296,6 +296,10 @@ func (_ *VPC) RenderTerraform(t *terraform.TerraformTarget, a, e, changes *VPC)
 		return err
 	}
 
+	if err := t.AddOutputVariable("vpc_ipv6_cidr_length", terraformWriter.LiteralFunctionExpression("local.vpc_ipv6_cidr_block == null ? null : tonumber", "regex(\".*/(\\\\d+)\", local.vpc_ipv6_cidr_block)[0]")); err != nil {
```

Comment:
```suggestion
	if err := t.AddOutputVariable("vpc_ipv6_cidr_length", terraformWriter.LiteralFunctionExpression("local.vpc_ipv6_cidr_length")); err != nil {
```

Reply:
This `AddOutputVariable` call is what defines `local.vpc_ipv6_cidr_length`. You can't define a local variable as being itself.